### PR TITLE
Add minutes from 2020-11-26

### PR DESCRIPTION
**Pull request by PeterJCLaw**
No actions as they've either already happened or weren't originated this meeting.

### CODE REVIEW

File: SR2021/2020-11-26.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+### Brief Kickstart retrospective
+
+#### Good
+
+ * 116 concurrent viewers!
+ * We shipped it, it was awesome
+ * It felt smooth
+ * Most teams were happy getting going with the simulator
+ * Teams did struggle with getting a Python set up
+    * Could we mitigate this by being clearer how to set that up?
```
Comment (sedders123) [inline SR2021/2020-11-26.md:29]: Should this be in 'Not so Good'?

File: SR2021/2020-11-26.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ * Discord seemed to work
+    * Being snappier with responses is good
+    * Relatively good analogue for being there in person
+    * Provides a good direct but still open comms channel to teams
+ * Really really rushed
+    * We should actually plan ahead (shock horror)
```
Comment (sedders123) [inline SR2021/2020-11-26.md:35]: Should this be in 'Not so Good'?

**Review by sedders123 — COMMENTED** [review]
All looks good thanks! Just two points that look like they should be in the other section of the retro

File: SR2021/2020-11-26.md (hunk 1 above)
Comment (PeterJCLaw) [reply SR2021/2020-11-26.md]: 02e72946c4b2f02247a2bfe223b6293e347ab964

File: SR2021/2020-11-26.md (hunk 2 above)
Comment (PeterJCLaw) [reply SR2021/2020-11-26.md]: 02e72946c4b2f02247a2bfe223b6293e347ab964

**Review by sedders123 — APPROVED** [review]

**Pull request merged**